entry.sh

```diff
@@ -16,7 +14,15 @@ elif [ ! -d "build" ]; then
     mkdir build
 fi
 
-cd build 
-cmake ..
-make -j
-make test
+if [[ $CROSS_COMPILE = true ]]; then
+    echo "Cross compiling for Raspberry Pi..."
+    cd build 
+    cmake  -DCMAKE_CXX_COMPILER=aarch64-rpi3-linux-gnu-g++ -DCURSES_LIBRARY=/usr/lib/x86_64-linux-gnu/libncurses.so -DCURSES_INCLUDE_PATH=/usr/include ..
+    make -j
```

Comment:
cos this is an arm container, you can run `make test` also right?

Reply:
pretty sure we can, but may be overkill as we presumably won't need to run the tests at this stage and will be fairly computationally expensive cause of emulation. Unless we want it as proof the cross-compiled code works.